bindings/pyroot_experimental/PyROOT/test/tfile.py

```diff
@@ -0,0 +1,58 @@
+import os
+import unittest
+
+import ROOT
+from libcppyy import SetOwnership
+
+
+class TFileUnits(unittest.TestCase):
```

Comment:
So far the strategy we have followed for the unit tests of the pythonizations is to divide them per pythonization, not per class as here. Example [here](https://github.com/root-project/root/pull/2476/commits/3a3941758449207c0ca8323700d4856129ab5723) for the pythonization of `TTree::SetBranchAddress`.

This way of splitting the tests also encourages the developer to write more extensive tests. I wonder if this strategy could be followed also here? E.g. for testing the pythonization of getting some object inside a file with the attribute syntax, could we have a separate file where we try different types of objects (e.g. not only histos but also trees or others?).

Reply:
Of course the strategy can be adopted also here. I am not sure about the additional coverage of writing different objects but we definitively can.

Reply:
Perhaps this is not the best example, but anyway it is cheap to extend the "use case" coverage, even if the code coverage is the same, just in case the code changes some day.